src/core/issues-notes.js

```diff
@@ -140,21 +152,30 @@ async function fetchIssuesFromGithub({ githubAPI }) {
     .map(elem => Number.parseInt(elem.dataset.number, 10))
     .filter(number => number);
   for (const issueNumber of issueNumbers) {
+    const issue = { title: "", number: issueNumber, status: "" };
```

Comment:
Why should we set this and use `Object.assign`, instead of just using `response.json()`?

Reply:
I wanted to avoid creating the object twice, but probably ok to just assign it.